webots_ros2_core/webots_ros2_core/__init__.py

```diff
@@ -11,3 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import logging
+logging.basicConfig()
+webots_ros2_core_logger = logging.getLogger('webots_ros2_core')
+webots_ros2_core_logger.warning(
+    ' This package will be removed with the release of Webots R2023a.'
+    ' Use "webots_ros2_driver" instead.'
```

Comment:
Why did you add a space in front of the text here?

Reply:
Just to make it more readable when the warning get printed in the terminal.

I saw some other warnings (not from our side) that do not have these spaces but I found it not very easy to read.

I can remove them if we want to be consistent with other warnings.